images/fcos-base-image/Containerfile

```diff
@@ -12,14 +12,14 @@ RUN set -x; arch=$(uname -m | sed 's/x86_64/amd64/;s/aarch64/arm64/'); cat /etc/
 # Also, installing nfs-utils will leave content in /var. `rm -rf /var/*` is a workaround until the packages are fixed.
 RUN rpm-ostree uninstall nfs-utils-coreos && rpm-ostree install nfs-utils && rm -rf /var/* && ostree container commit
 
-RUN set -x; PACKAGES_INSTALL="bridge-utils conntrack-tools curl fping iftop iputils iproute mtr nethogs socat chrony \
-                              targetcli net-tools bind-utils iperf iperf3 iputils mtr ethtool tftp wget ipmitool gawk \
-                              htop ncdu procps strace iotop subversion git git-lfs gnupg2 lldpd openssl openvpn rsync tcpdump  \
-                              nmap nmap-ncat krb5-workstation qemu-kvm qemu-user-static libvirt virt-manager virt-install lshw \
-                              sudo screen telnet unzip util-linux-user ignition  zsh nmap-ncat socat python3-pip man pciutils  \
+RUN set -x; PACKAGES_INSTALL="bridge-utils conntrack-tools curl fping iftop iputils iproute mtr nethogs socat chrony iperf \
+                              iperf3 iputils mtr ethtool tftp wget ipmitool gawk targetcli net-tools bind-utils htop ncdu \
+                              procps strace iotop subversion git git-lfs gnupg2 lldpd openssl openvpn rsync nmap nmap-ncat zsh \
+                              krb5-workstation qemu-kvm qemu-user-static libvirt virt-manager virt-install crypto-policies-scripts \
+                              sudo screen telnet unzip util-linux-user ignition lshw tcpdump socat python3-pip man pciutils  \
                               skopeo jq vim neovim inotify-tools firewall-config openvswitch NetworkManager-ovs tree sshpass"; \
     rpm-ostree install $PACKAGES_INSTALL \
-    && ln -s /usr/bin/ld.bfd /usr/bin/ld \
+    && rm -f /usr/sbin/ovs-vswitchd \
```

Comment:
Can we use ln -sf instead of deleting first?

Reply:
done